package.json

```diff
@@ -83,10 +84,11 @@
     "build": "npm-run-all --parallel build:js build:css --serial build:gzip size",
     "clean": "rm -rf packages/*/lib packages/@uppy/*/lib && rm -rf packages/uppy/dist",
     "lint:fix": "npm run lint -- --fix",
+    "lint:companion": "eslint --ignore-path packages/@uppy/companion/.eslintignore -c packages/@uppy/companion/.eslintrc.json packages/@uppy/companion/.",
```

Comment:
shouldn't eslint use the `.eslintrc.json` in that directory when we run it on the entire project? :thinking:

Reply:
from my observation, it seems it merges both options (which is not desired). Try `node_modules/.bin/eslint --print-config packages/@uppy/compan
ion`

Reply:
Oh indeed, you're right. Looks like I've been doing `extends: "../../.eslintrc.json"` in my projects for no reason all this time :stuck_out_tongue:

There is a `root: true` option that we could set in companion's eslintrc that should do the trick: https://eslint.org/docs/user-guide/configuring#using-configuration-files

Reply:
fixed